src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -58,6 +58,7 @@ export const App = ({ displayToggle, toggleLoginModal }) => {
       apiRequest('/form1095_bs/available_forms')
         .then(response => {
           if (response.errors || response.availableForms.length === 0) {
```

Comment:
IMO, this condition is not quite right. The backend can return response.errors for authentication or authorization issues. Those don't happen because we prevent the calls from being made. But I think a more accurate way to approach this would probably be to show the appropriate not authenticated/authorized errors if those ever do occur and a system error for any other errors. I don't think there's a case where the existence of errors implies that the user doesn't have forms.

Reply:
I'm I'm following you, would it make sense to remove the first condition (`response.errors`)? There's a `catch` block after this that seems to handle errors itself.